clang/cmake/caches/Fuchsia-stage2.cmake

```diff
@@ -31,6 +31,7 @@ set(LLVM_STATIC_LINK_CXX_STDLIB ON CACHE BOOL "")
 set(LLVM_USE_RELATIVE_PATHS_IN_FILES ON CACHE BOOL "")
 set(LLDB_ENABLE_CURSES OFF CACHE BOOL "")
 set(LLDB_ENABLE_LIBEDIT OFF CACHE BOOL "")
+set(LIBCXX_INCLUDE_BENCHMARKS OFF CACHE BOOL "")
```

Comment:
This needs to be set for the target, that is line 184 for Linux.

Reply:
Thanks - Done. I looked at the pattern of where we set fellow `LIBCXX_...` macros and made adjustments in in Fuchsia.cmake as well.

Reply:
Also - PTAL @ https://github.com/llvm/llvm-project/pull/115361#issuecomment-2480526628.